src/extension.ts

```diff
@@ -78,10 +79,18 @@ function getPRsPerSHAs() {
   watermelonPanel.currentPanel?.doRefactor({
     command: "loading",
   });
+
+  // Splice the array to avoid the 256 character restriction on the query to the GitHub API
+  if (arrayOfSHAs.length > 22) {
+    arrayOfSHAs.splice(22, arrayOfSHAs.length - 22);
+  }
+
+  let joinedArrayOfSHAs = arrayOfSHAs.join();
```

Comment:
Had to check, `join` joins with a ","

const elements = ['Fire', 'Air', 'Water'];

console.log(elements.join());
// output: "Fire,Air,Water"``